src/ntfs/read.rs

```diff
@@ -110,7 +111,10 @@ pub fn manage_fares_v1(collections: &mut Collections, base_path: &path::Path) ->
     let file_od_fares = "od_fares.csv";
     let file_fares = "fares.csv";
 
-    if !base_path.join(file_prices).exists() || !base_path.join(file_od_fares).exists() {
+    if !base_path.join(file_prices).exists()
```

Comment:
I'm not sure I understand this `if` test. We do not want to write Fare V1 if:
1. We do not have Fare V1 in the input
2. We do have Fare V2 in the model

I thought every time we do have Fare V2, we want to write V1. And if we do not have Fare V2 in the model, we want to write V1 only if it's in the input NTFS... so basically the opposite of this test. Just feels like I'm missing something.